pkgs/development/python-modules/pytest-flask/default.nix

```
@@ -12,14 +12,16 @@ buildPythonPackage rec {
   doCheck = false;
 
   buildInputs = [
```

I know it's not your change, but may as well while we are addressing this
```suggestion
  checkInputs = [
```

---

looks like they add pytest in the main dependencies:
```
Finished executing setuptoolsBuildPhase
installing
Executing pipInstallPhase
/build/pytest-flask-1.0.0/dist /build/pytest-flask-1.0.0
Processing ./pytest_flask-1.0.0-py3-none-any.whl
Requirement already satisfied: Werkzeug>=0.7 in /nix/store/61pdl81yb47bz8bf62jfnbx31mbivivg-python3.8-Werkzeug-1.0.1/lib/python3.8/site-packages (from pytest-flask==1.0.0) (1.0.1)
Requirement already satisfied: Flask in /nix/store/mmi6limfihgxszmz7kka24sbzhkdbz3n-python3.8-Flask-1.1.2/lib/python3.8/site-packages (from pytest-flask==1.0.0) (1.1.2)
ERROR: Could not find a version that satisfies the requirement pytest>=5.2 (from pytest-flask==1.0.0) (from versions: none)
ERROR: No matching distribution found for pytest>=5.2 (from pytest-flask==1.0.0)
builder for '/nix/store/r7jjp5fn0b4lsg8wyw3vm90xjdh7l59q-python3.8-pytest-flask-1.0.0.drv' failed with exit code 1
error: build of '/nix/store/r7jjp5fn0b4lsg8wyw3vm90xjdh7l59q-python3.8-pytest-flask-1.0.0.drv' failed
```
https://github.com/pytest-dev/pytest-flask/blob/master/requirements/main.txt

---

for pytest flask, that makes sense

---

actually, this should be in propagatedBuildInputs, since it's a plugin trying to propagate the test runner